ui/src/views/offering/AddComputeOffering.vue

```diff
@@ -192,7 +192,7 @@
               rules: [{ required: true, message: $t('message.error.required.input') },
                       {
                         validator: (rule, value, callback) => {
-                          if (value && (isNaN(value) || value <= 0)) {
+                          if (value && isNaN(value)) {
```

Comment:
```suggestion
                          if (value && (isNaN(value) || value < 0)) {
```
Guard for negative values